### PR TITLE
Feature tf bias fix

### PR DESCRIPTION
**Pull request by aniktash**

### CODE REVIEW

File: tests/tensorflow/simple-maxpool-conv2d-relu-dense.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+arch: ai85net5
+dataset: simple-maxpool-con2d-dense
```
Comment (vicloginov) [inline tests/tensorflow/simple-maxpool-conv2d-relu-dense.yaml:2]: There is no ReLu operation but name of file has Relu.
No newline at end of file.

**Review by rotx-maxim — APPROVED** [review]

File: tests/tensorflow/simple-avgpool-conv2d-relu-dense.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+arch: ai85net5
+dataset: simple-avgpool-con2d-dense
```
Comment (vicloginov) [inline tests/tensorflow/simple-avgpool-conv2d-relu-dense.yaml:2]: There is no ReLu operation but name of file has Relu.
No newline at end of file.

**Review by vicloginov — COMMENTED** [review]
There is no ReLu operation in some YMAL files but file name has Relu.

**Comment by aniktash** [comment]
> There is no ReLu operation in some YMAL files but file name has Relu.

PR is merged, I'll update test names in next round

**Pull request merged**